README.md

```diff
@@ -1,12 +1,89 @@
 # cdnExtension
```

Comment:
Please update it to EasyCDN

Reply:
Okay I'm on it

Reply:
I have created the pull request I'm waiting for your approval